Aero_Sub_DAQ/Air_pressure_sensor_code/air_pressure_code_v2.ino

```diff
@@ -0,0 +1,143 @@
+/*
+* Air Pressure Sensor
+* Authors:        Allison Pham         Embedded Firmware Team
+*                 Srinidhi Malladi
+*                 William Everson
+* 
+* Email:          willreverson@gmail.com
+* 
+* (c) 2022 Dallas Formula Racing - Formula SAE
+* GPL-3.0 License
+*/
+
+#include <Wire.h>
+#include <SPI.h>
+#include <SD.h>
+
+#define I2C_ADDRESS 0x29
+//initialize global variables
+const int kSwitchPin = 7, kChipSelect = 10;
+const float kTwo24 = 16777216.00;
+uint8_t command[8], StatusByte, cmdbyte;
+uint32_t sensor_output_bytes[7];
+int count;
+float time;
+String file_name = "aerodata.csv";
+File dataFile;
+bool switchUsed;

```

Comment:
For global variables, add `static` before the type declaration
For constant globals, add `static constexpr`